recipes/example/meta.yaml

```diff
@@ -45,6 +45,7 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
   license: MIT
+  license_family: MIT
```

Comment:
So I like the positioning of this, but it makes it seem required, which is not something we have agreed on. Could we maybe add a one line comment above the `license_family` that notes this is an optional field?